src/allmydata/storage/lease_schema.py

```diff
@@ -56,7 +56,7 @@ def _hash_secret(cls, secret):
         """
         Hash a lease secret for storage.
         """
-        return blake2b(secret, digest_size=32, encoder=RawEncoder())
+        return blake2b(secret, digest_size=32, encoder=RawEncoder)
```

Comment:
Huh this is a little worrying....  I guess it means there are no tests for hashed lease writing?  Or, huh.  It seems to work either way, somehow.  Oh.  because its interface is a staticmethod.  Alrighty then, fine.